### ISSUE TITLE
Могу ли я добывать ect и rvn одновременно на одном GPU?

### ISSUE THREAD

**Issue by TheIlyaSteep** [UnamSanctam/SilentCryptoMiner#2238]
Могу ли я добывать ect и rvn одновременно на одном GPU?
![1](https://github.com/UnamSanctam/SilentCryptoMiner/assets/42769532/cc1443af-f258-4801-b249-e1b6c20c0810)

**Comment by UnamSanctam** [comment]
Yes though it's not really recommended, since both will be using the GPU resources.
